client/src/styles/global.scss

```diff
@@ -122,7 +124,7 @@ components include:
 
   span {
     // make sure the open close chevron is colored correctly
-    color: $primary-color;
+    color: $primary-color; // Todo color: remove this global
```

Comment:
I wonder if this can be removed now? I think it was from a time when the top nav was inverted (dark themed).